app/sagas/cpacpy.js

```diff
@@ -174,31 +193,31 @@ function* connect({ scheduler: id }) {
   }
 }
 
-function* callScheduler({
+function* callAny({
   scheduler: id,
   method='GET',
   endpoint,
   data,
   response: { success, error },
   headers = {}
 }) {
-
   const scheduler = yield selectSaga(selectScheduler(id))
+  const url = `http://${scheduler.get('address')}${endpoint}`
 
   const success_return = (data, headers) =>
     success instanceof Function ?
       success(data, headers) :
       cpacpyCallSuccess(scheduler, success, data, headers)
 
-  const error_return = (exception, headers) =>
```

Comment:
What's the rationale to not forward the response headers?

Reply:
Because in some cases, there's no response headers for an error return, thus the parameter was removed.

Reply:
Ok!